packages/mgt/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -110,6 +111,7 @@ export class MgtTodo extends MgtTasksBase {
     this._tasks = [];
     this._loadingTasks = [];
     this._isLoadingTasks = false;
+    this._strings = strings;
```

Comment:
You shouldn't have to do this if you handle it in baseComponent with the strings getter

Reply:
updated with new getters and setters:

a079450